docs/2023/Sunbird ED/Implement Independent SSO Flow on NPM Package/updates/2023-08-25.md

```diff
@@ -4,13 +4,18 @@ author: Om Tanmaya Pati
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- 🚀 I will be completing my code and eagerly awaiting a code review from my mentor. 🕒
+-  📝 By the end of this week, I'll have completed the documentation phase of the project.
+- 🔧 Currently addressing: There's an ongoing issue with the Sunbird Ed Portal that requires attention.
+- 
```

Comment:
Is there any xtra point which you forgot? if not you can remove this

Reply:
Actually i  forgot to delete. No issue i will change it next week pr